packages/core/src/components/Toggle/types.ts

```diff
@@ -26,4 +27,8 @@ export interface ToggleProps extends InputProps {
     labelWeight?: FontWeights;
     /** Use if required in any form for example user needs to check 'I Agree' */
     required?: boolean;
+    /** Icon to show when toggle is in OFF state */
+    iconRight?: ReactElement;
+    /** Icon to show when toggle is in ON state */
+    iconLeft?: ReactElement;
```

Comment:
Can we use OnIcon and OffIcon to be more clear? Also this is the naming convention in the styled file, we can keep it consistent here